packages/node/src/lib/webhook-verify.ts

```diff
@@ -0,0 +1,25 @@
+const crypto = require('crypto');
+
+interface WebhookBody {
+  email: String;
+}
+
+export function verify(body: WebhookBody, signature: string, secret: string): WebhookBody {
```

Comment:
How would you feel about calling this `verifyWebhook()` instead? verify on it's own is a little generic.